README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
This seems rather counter intuitive. I feel as though it should be a key combination of left alt + up arrow + W + 4 + f9 + L + L - 2